multy_core/src/bitcoin/bitcoin_transaction.cpp

```diff
@@ -521,7 +521,7 @@ BinaryDataPtr BitcoinTransaction::serialize()
 }
 
 template <typename T>
-void BitcoinTransaction::serialize_to_stream(T* stream) const
+void BitcoinTransaction::serialize_to_stream(T* stream, IncludeChangeDestination with_change_destinations) const
```

Comment:
Please rename variable `with_change_destinations` to something like `destinations_to_use`.